### PR TITLE
Update use case page ui

### PR DESCRIPTION
**Pull request by YayunHuang**
Ref: MUP-127

https://github.com/user-attachments/assets/5fe11eff-9ff4-4e72-97f1-ab31cf2fc8a7

### CODE REVIEW

File: src/styles/use_case.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  .col-12 {
+    padding: 20px !important;
+  }
+}
+
+@media (max-width: 768px) {
+  .use-case-page {
+    .title,
+    p {
+      text-align: center !important;
```
Comment (pkong-ds) [inline src/styles/use_case.css]: I think justified looks better than center, @lilllian31cat any thoughts?

| justified    | center    |
|-------|-------|
| image | image |

File: src/styles/use_case.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    font-size: 32px;
+    font-weight: bold;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
+
+  .col-12 {
+    padding: 20px !important;
```
Comment (pkong-ds) [inline src/styles/use_case.css:19]: why `!important`, which style is it overriding?

File: src/styles/use_case.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+@media (max-width: 768px) {
+  .use-case-page {
+    .title,
+    p {
+      text-align: center !important;
+    }
+
+    .col-12 {
+      padding: 16px !important;
```
Comment (pkong-ds) [inline src/styles/use_case.css:30]: same

File: src/pages/use_case.astro [hunk 4]
```diff
@@ -9,57 +10,86 @@ import BaseLayout from "../layouts/BaseLayout.astro";
...
+          </p>
+        </div>
+        <div></div>
+        <div class="col-12 col-md-6 col-lg-4 use-case-card">
+          <div class="icon">
+            <img src="/Images/use_case_1.svg" />
+          </div>
+          <h2>App Design Presentation</h2>
+          <p>
+            Present your web/ app designs using real device canvas, impress your
```
Comment (pkong-ds) [inline src/pages/use_case.astro]: extra white space

File: src/pages/use_case.astro [hunk 5]
```diff
@@ -9,57 +10,86 @@ import BaseLayout from "../layouts/BaseLayout.astro";
...
+            drag-and-drop tool.
+          </p>
+        </div>
+        <div class="col-12 col-md-6 col-lg-4 use-case-card">
+          <div class="icon">
+            <img src="/Images/use_case_2.svg" />
+          </div>
+          <h2>Responsive Design Presentation</h2>
+          <p>
+            Compare the looks of your web/ app designs on different canvas to
```
Comment (pkong-ds) [inline src/pages/use_case.astro]: extra white space

File: src/styles/use_case.css [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+  h2 {
+    font-size: 20px;
+    font-weight: bold;
+    margin-bottom: 4px;
+  }
+}
+
+@media (max-width: 768px) {
+  .use-case-card:not(.align-left) {
```
Comment (pkong-ds) [inline src/styles/use_case.css]: Why not center-align for `Still need a reason?`? Below looks inconsistent

image

File: src/styles/use_case.css (hunk 2 above)
Comment (YayunHuang) [reply src/styles/use_case.css:19]: @pkong-ds to override bootstrap `col-12` default padding

File: src/styles/use_case.css (hunk 6 above)
Comment (YayunHuang) [reply src/styles/use_case.css]: `Still nee a reason` part is designed to be left aligned
image

**Comment by YayunHuang** [comment]
@pkong-ds updated as design:
Iphone XR:
localhost_3000_use_case (1)
browser:
localhost_3000_use_case
Ipad Air:
localhost_3000_use_case(iPad Air)

**Comment by pkong-ds** [comment]
LGTM

| mobile         | desktop                                   | tablet                                   |
|---------------------|------------------------------------------|------------------------------------------|
| localhost_3000_use_case (1)          | localhost_3000_use_case | localhost_3000_use_case(iPad Air)   |

**Review by pkong-ds — APPROVED** [review]

**Pull request merged**